ipfs/env.sh

```diff
@@ -36,6 +36,8 @@ all_ipfs_repo=/ipfs/ipfs_master/repo
 
 # The datastore lookup cache - 512KiB
 all_ipfs_bloom_size=524288
+# 128MiB for storage hosts, should be good for up to about 200M objects
+storage_ipfs_bloom_size=134217728
```

Comment:
This needs to be prefixed with hostnames instead, since provsn doesn't have grouping